### PR TITLE
update dashboard for authentication

### PR DESCRIPTION
**Pull request by jgsqware**
Signed-off-by: Julien Garcia Gonzalez <julien@giantswarm.io>

With RBAC enabled, Dashboard is currently not accessible without authentication.
I've added a section to create Service Account Token for Cluster Admin and a section for restricted access.

### CODE REVIEW

File: content/guides/install-kubernetes-dashboard/index.md [hunk 1]
```diff
@@ -25,15 +27,108 @@ $ kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master
 
 Once the pod is running you can open Dashboard at `https://api.<cluster-id>.k8s.gigantic.io/ui`.
 
-*Note*: The above URL uses your Kubernetes API to proxy to the service. As the API is guarded with your credentials, you need to [set them up in your system](/guides/accessing-services-from-the-outside/) (and/or browser). We do not recommend to set up an Ingress for the Dashboard at this time, as Dashboard currently does not support any kind of authentication and thus your cluster would be open to everyone.
+*Note*: The above URL uses your Kubernetes API to proxy to the service. As the API is guarded with your credentials, you need to [set them up in your system](/guides/accessing-services-from-the-outside/) (and/or browser).
```
Comment (puja108) [inline content/guides/install-kubernetes-dashboard/index.md]: Why did you remove that recommendation, I would still not recommend it, you are opening up a 2nd way into the cluster without real control.

Comment (puja108) [reply content/guides/install-kubernetes-dashboard/index.md]: My point is: even if you just get the auth page, for some clients we might have k8s api behind vpn, bastion, what have you, a user then opening up the dashboard to public would create a non-official workaround to access k8s api.

File: content/guides/install-kubernetes-dashboard/index.md [hunk 2]
```diff
@@ -25,15 +27,108 @@ $ kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master
...
+
+After login, the user can only see `namespaces` and `pods`.
+
+![Kubernetes Dashboard Limited user](/img/dashboard-pod-viewer.png)
+
+## Further Reading
 
-Check out the [official user guide](http://kubernetes.io/docs/user-guide/ui/) for details on how to use Dashboard.
+- [Official User Guide](http://kubernetes.io/docs/user-guide/ui/)
+- [On Securing the Kubernetes Dashboard](https://blog.heptio.com/on-securing-the-kubernetes-dashboard-16b09b1b7aca)
```
Comment (puja108) [inline content/guides/install-kubernetes-dashboard/index.md]: <3 that we added this link

File: content/guides/install-kubernetes-dashboard/index.md (hunk 1 above)
Comment (jgsqware) [reply content/guides/install-kubernetes-dashboard/index.md]: What about something like this so:

```

Comment (puja108) [reply content/guides/install-kubernetes-dashboard/index.md]: works for me, but make it `the K8s API`

could add that if you want to do that you should make sure your security department is ok with that and then still check out the blog post below ;)

Comment (jgsqware) [reply content/guides/install-kubernetes-dashboard/index.md]: done
